test/lightning_web/live/hooks_test.exs

```diff
@@ -1,5 +1,5 @@
 defmodule LightningWeb.HooksTest do
-  use LightningWeb.ConnCase, async: true
+  use LightningWeb.ConnCase, async: false
```

Comment:
Oh interesting, why do you think this was necessary?

Reply:
For some flaky let's see